lib/cards/SkeletonCard.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+
+  <!--
+    Required props set to avoid console validation errors.
+    Their logic is disabled in skeleton mode.
+  -->
+  <KCard
+    :to="{ path: null }"
+    :headingLevel="2"
+    isSkeleton
+    aria-hidden="true"
+    class="skeleton-card"
+    :style="{ height: height }"
+    :orientation="orientation"
+    :thumbnailDisplay="thumbnailDisplay"
+    :thumbnailAlign="thumbnailAlign"
+  >
+    <template #title>
+      <span
+        class="title-placeholder"
+        :style="{ backgroundColor: $themePalette.grey.v_50 }"
```

Comment:
I just merged #775 into develop, so these two uses of theme palette will need to be updated to `v_100` :).

Reply:
Thank you, @AlexVelezLl, done.